lib/carin_for_blue_button_test_kit/custom_groups/visual_inspection_and_attestation/v200_server/00_authorization_group/attestation_test_requirement_48.rb

```diff
@@ -0,0 +1,50 @@
+module CarinForBlueButtonTestKit
+  class AttestationTestCarinV2Requirement48_57 < Inferno::Test
+    title 'Secures data'
+    description <<~DESCRIPTION
+      The Health IT Module 
+      - Enables members to direct the communication of CARIN4BB data through authenticated, authorized, and secure channels.
```

Comment:
Looks like this line and requirement 48 are in `authentication.rb`. Do we need both?

Reply:
yes - one is only in the client suite and one is only in the server suite